### ISSUE TITLE
 gnt-instance(8) / grow-disk does not mention template {file,sharedfile}

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#313]
Originally reported of Google Code with ID 257.

```
What software version are you running? Please provide the output of "gnt-
cluster --version" and "gnt-cluster version".
# gnt-cluster --version
gnt-cluster (ganeti v2.6.0) 2.6.0
# gnt-cluster version
Software version: 2.6.0
Internode protocol: 2060000
Configuration format: 2060000
OS api version: 20
Export interface: 0

<b>What distribution are you using?</b>
# cat /etc/SuSE-release
SUSE Linux Enterprise Server 11 (x86_64)
VERSION = 11
PATCHLEVEL = 2

the manpage gnt-instance(8) section grow-disk says: "This is only possible for instances having a plain, drbd or rbd disk template". However trying with template sharedfile also works:

$ gnt-instance info somevm.com
...
  Disk template: sharedfile
  Disks:
    - disk/0: file, size 10.0G
      access mode: rw
      logical_id:  ['loop', '/srv/ganeti/shared-file-storage/somevm.com/disk0']
      on primary:  /srv/ganeti/shared-file-storage/somevm.com/disk0 (N/A:N/A)
    - disk/1: file, size 15.0G
      access mode: rw
      logical_id:  ['loop', '/srv/ganeti/shared-file-storage/somevm.com/disk1']
      on primary:  /srv/ganeti/shared-file-storage/somevm.com/disk1 (N/A:N/A)

# gnt-instance grow-disk somevm.com 1 1G
Fri Aug 10 10:08:50 2012 Growing disk 1 of instance 'somevm.com' by 1.0G to 16.0G
Fri Aug 10 10:08:55 2012  - INFO: Waiting for instance somevm.com to sync disks.
Fri Aug 10 10:08:55 2012  - INFO: Instance somevm.com's disks are in sync.

$ gnt-instance info somevm.com
...
    - disk/1: file, size 16.0G
      access mode: rw
      logical_id:  ['loop', '/srv/ganeti/shared-file-storage/somevm.com/disk1']
      on primary:  /srv/ganeti/shared-file-storage/somevm.com/disk1 (N/A:N/A)

# ls -alh /srv/ganeti/shared-file-storage/somevm.com/disk1
-rw-r--r-- 1 nobody nobody 16G Aug 10 10:09 /srv/ganeti/shared-file-storage/somevm.com/disk1

Please update the documentation.

TIA, Sascha.
```
Originally added on 2012-08-10 08:15:52 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2012-12-13 15:57:31 +0000 UTC.

Changed State: Started

**Comment by Ganeti-Issues-Migrator** [comment]
```
Fixed, will be in 2.6.2.
```
Originally added on 2012-12-13 17:46:56 +0000 UTC.

Added Labels: Type-Documentation
Changed State: Fixed
Added to Milestone: Release2.6

**Comment by Ganeti-Issues-Migrator** [comment]
```
Marking as released the bugs fixed in 2.6.2.
```
Originally added on 2013-01-11 12:13:05 +0000 UTC.

Changed State: Released
